### PR TITLE
Replace booster by example

### PR DESCRIPTION
**Pull request by aureamunoz**
Related to: SB-1345

### CODE REVIEW

File: json.json [hunk 1]
```diff
@@ -0,0 +1,1551 @@
+[
```
Comment (gytis) [inline json.json]: What's the purpose of this file?

Comment (aureamunoz) [reply json.json]: :-S no idea

Comment (gytis) [reply json.json]: looks like newly added in this PR

Comment (aureamunoz) [reply json.json]: :-o may be my git aa added it :-S. I'll check!

Comment (aureamunoz) [reply json.json]: I've deleted the file

**Review by gytis — APPROVED** [review]

**Pull request merged**